chat-components/src/main/java/com/pubnub/components/chat/ui/component/input/MessageInputExt.kt

```diff
@@ -24,9 +19,7 @@ fun MessageInput(
     initialText: String = "",
     placeholder: String = stringResource(id = R.string.type_message),
     typingIndicatorEnabled: Boolean = false,
-    typingIndicatorContent: @Composable ColumnScope.(List<TypingUi>) -> Unit = { typing ->
-        TypingIndicatorContent(typing)
-    },
+    typingIndicator: @Composable () -> Unit = { TypingIndicator() },
```

Comment:
Should still use `ColumnScope` to allow custom modifications based on root layout

Reply:
The name should stay too. `typingIndicator` suggest that the instance of indicator will be passed.